### PR TITLE
[Hotfix] Fix division by 0 error

### PR DESCRIPTION
**Pull request by fleupold**
0xfa3a5ba1864c4567ac77d50ecd91a2aae92b650d placed an order with 0 numerator causing division by 0 in our Orderbook.

To see the order, run the following query in the [rinkeby graph](https://thegraph.com/explorer/subgraph/gnosis/dfusion-rinkeby)

```
{
  orders(where: {priceNumerator: 0, buyToken: "4", sellToken: "1",owner: "0xfa3a5ba1864c4567ac77d50ecd91a2aae92b650d"}, orderBy: fromBatchId, orderDirection: desc) {
    id
    sellToken {
      id
    }
    buyToken {
      id
    }
    priceNumerator
    priceDenominator
    fromEpoch
  }
}
```

The stacktrace:

```
Error: Assertion failed
    at assert (/Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/node_modules/bn.js/lib/bn.js:6:21)
    at BN.divmod (/Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/node_modules/bn.js/lib/bn.js:2418:5)
    at BN.div (/Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/node_modules/bn.js/lib/bn.js:2514:17)
    at Fraction.reduce (/Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/build/common/src/fraction.js:28:41)
    at new Fraction (/Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/build/common/src/fraction.js:13:14)
    at Fraction.mul (/Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/build/common/src/fraction.js:38:16)
    at /Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/build/common/src/orderbook.js:324:44
    at Array.map (<anonymous>)
    at invertPricePoints (/Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/build/common/src/orderbook.js:320:49)
    at Orderbook.inverted (/Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/build/common/src/orderbook.js:79:23)
    at Orderbook.transitiveClosure (/Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/build/common/src/orderbook.js:178:40)
    at /Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/build/common/src/orderbook.js:278:32
    at Map.forEach (<anonymous>)
    at transitiveOrderbookRecursive (/Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/build/common/src/orderbook.js:273:16)
    at Object.transitiveOrderbook (/Users/felixleupold/Gnosis/dex-price-estimator/node_modules/@gnosis.pm/dex-contracts/build/common/src/orderbook.js:258:12)
    at router.get (/Users/felixleupold/Gnosis/dex-price-estimator/build/index.js:67:40)
```

The fix is to ignore orders that have weird prices (numerator or denominator being 0)

### Test Plan

```
export ETHEREUM_NODE_URL=https://blue-parity.rinkeby.gnosisdev.com
yarn start
```

once orderbook is fetched query

`http://localhost:8080/api/v1/markets/4-7?atoms=true&hops=1`

and see no errors

### CODE REVIEW

**Review by e00E — APPROVED** [review]
Seems fine, just wondering if there is a need to special case unlimited orders.

**Comment by fleupold** [comment]
> wondering if there is a need to special case unlimited orders.

What is your concern about them? They have valid prices so they shouldn't cause numerical issues. And since the orderbook is not updated based on events in memory it will read the updated (or constant) sell amounts from chain.

**Review by nlordell — APPROVED** [review]

**Review by bh2smith — APPROVED** [review]

File: src/orderbook_fetcher.ts [hunk 1]
```diff
@@ -84,19 +84,22 @@ async function updateOrderbooks(contract: BatchExchangeViewer, pageSize: number,
 }
 
 function addItemToOrderbooks(orderbooks: Map<string, Orderbook>, item: OrderBN) {
+  const MIN_TRADEABLE_VOLUME = new Fraction(10000, 1)
```
Comment (anxolin) [inline src/orderbook_fetcher.ts:87]: Maybe this better off in a constant files?

**Review by anxolin — COMMENTED** [review]
👍

**Pull request merged**